src/generator.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+function generateNumber() {
+  const digits = [];
```

Comment:
It's excellent idea to use there an array, but it would be better use here `new Set`. [new Set by MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) `Set` could help you store unique values, and you don't need to write additional checking if value exist in your array(9 str in this file).

And in a future, when you need to check if value exist in your `Set` you can use `.has()` method, because it checks directly, than `includes()` that checks values one by one.